### PR TITLE
[hlf-couchdb] Update Owners + bug fixes and best-practices

### PR DESCRIPTION
**Pull request by alexvicegrab**
**What this PR does / why we need it**:

Add OWNERS file
Update logo
Fix typo
Update persistence in line with Helm best practices.

**Special notes for your reviewer**:

None

### CODE REVIEW

File: stable/hlf-couchdb/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ $ helm install stable/hlf-couchdb --name cdb1 --set couchdbUsername=couchdb,couc
 
 The above command creates a CouchDB user named `couchdb` with password `secretpassword`.
 
-Alternatively, a YAML file can be provided while installing the chart. This file specifies values to override those provided in the default values.yaml. For example,
+Alternatively, a YAML file can be provided while installing the chart. This file specifies values to override those provided in the defualt values.yaml. For example,
```
Comment (davidkarlsen) [inline stable/hlf-couchdb/README.md]: typo

Comment (alexvicegrab) [reply stable/hlf-couchdb/README.md]: Thank you, will fix now.

Comment (alexvicegrab) [reply stable/hlf-couchdb/README.md]: Done

**Comment by alexvicegrab** [comment]
@prydonius, @sameersbn, @prydonius, please review when you have the chance.

**Pull request merged**